src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -43,6 +44,9 @@
         package removal but accepts an optional argument for the number of
         days.
     """)
+mask_opts.add_argument(
+    '-b', '--bug', dest='bugs', action='append', type=arghparse.positive_int,
+    help='reference bug in the mask comment')
```

Comment:
I forgot to note, you may consider adding a `docs` parameter similar to those above which is the long description used for the option when auto-generating the related man page.